compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/ComposeSceneMediator.desktop.kt

```diff
@@ -240,6 +241,12 @@ internal class ComposeSceneMediator(
         override fun keyTyped(event: KeyEvent) = onKeyEvent(event)
     }
 
+    private val eventListener = if (eventListener != null) {
+        AwtEventListeners(OnlyValidPrimaryMouseButtonFilter, eventListener)
+    } else {
+        OnlyValidPrimaryMouseButtonFilter
+    }
+
```

Comment:
Not a big deal, but the repeating of `OnlyValidPrimaryMouseButtonFilter` is not a good look, and a potential source of bugs.